File: SEP3_T1/Shared/Domain/DTOs/UpdateCommentDTO.cs
namespace DTOs.DTOs;

public class UpdateCommentDTO
{
    public string body { get; set; }
    public int commentId { get; set; }
    
    public UpdateCommentDTO(int commentId, string body)
    {
        this.body = body;
        this.commentId = commentId;
    }
}

File: SEP3_T2/grpc-springboot/src/main/java/sep3/dto/UpdateCommentDTO.java
package sep3.dto;

public class UpdateCommentDTO
{
  private String body;
  private int commentId;

  public UpdateCommentDTO(int commentId, String body)
  {
    this.body = body;
    this.commentId = commentId;
  }

  public int getCommentIdId()
  {
    return commentId;
  }

  public String getBody()
  {
    return body;
  }
}
